app/components/data-download.tsx

```diff
@@ -28,16 +27,16 @@ import {
 } from "../graphql/query-hooks";
 import { Icon } from "../icons";
 import { useLocale } from "../locales/use-locale";
-
-const EXTENTS = ["visible", "all"] as const;
-type Extent = typeof EXTENTS[number];
+import Flex from "./flex";
 
 const FILE_FORMATS = ["csv", "xlsx"] as const;
 export type FileFormat = typeof FILE_FORMATS[number];
 
-const OPTIONS_TO_RENDER = EXTENTS.flatMap((extent) =>
-  FILE_FORMATS.map((fileFormat) => ({ extent, fileFormat }))
-);
+type PreparedData = {
+  columnKeys: string[];
+  data: Observation[];
+  sparqlEditorUrl: Maybe<string> | undefined;
```

Comment:
Maybe add a "?" `sparqlEditorUrl?: Maybe` ? This way we do not have to add `sparqlEditorUrl: undefined`